lab_3_keywords_textrank/main.py

```diff
@@ -136,7 +147,11 @@ def encode(self, tokens: tuple[str, ...]) -> Optional[tuple[int, ...]]:
                 sequence of integer tokens
         In case of empty tokens input data, None is returned
         """
-        pass
+        if not tokens:
+            return None
+        self._learn_indices(tokens)
+        tokens_list = [self._word2id[key] for token in tokens for key in self._word2id if token == key]
```

Comment:
why do you create list and than cast it to tuple? Can you create tuple immediately?

Reply:
done